collect_app/src/main/java/org/odk/collect/android/external/ExternalSQLiteOpenHelper.java

```diff
@@ -263,4 +265,23 @@ private void onProgress(String message) {
             formLoaderTask.publishExternalDataLoadingProgress(message);
         }
     }
+
+    /**
+     * replaceByteOrderMark
+     *
+     * Removes the presence of a Byte Order Mark from the initial UTF-8 String
+     *
+     * Fixes issue: https://github.com/opendatakit/collect/issues/1695
```

Comment:
I would remove the link fo issue since we generally don't do that in other places. If someone needs to find the issue it's easy using GIT.

Reply:
Agreed